test/common/network/dns_impl_test.cc

```diff
@@ -364,17 +364,21 @@ TEST_P(DnsImplTest, LocalLookup) {
   }
 
   if (GetParam() == Address::IpVersion::v6) {
-    EXPECT_EQ(nullptr, resolver_->resolve("localhost", DnsLookupFamily::Auto,
+    std::string error_msg = "Synchronous DNS IPv6 localhost resolution failed. Please add ::1 "
```

Comment:
"Please verify localhost resolves to ::1 in /etc/hosts, since this misconfiguration is a common cause of these failures.".